Update reference Kafka consumer/producer

jerrypnz opened the pull request.

Commit:
Update reference Kafka consumer/producer

jackhopner commented:
lgtm once changes from https://github.com/movio/movio-apidoc-generator/pull/42 are completed.

Commit:
Review changes

- Updated the reference spec to make the topic contain all the possible
  variables.
- Re-generated the code

jackhopner commented on apidoc.json:
```diff
@@ -101,7 +101,7 @@
           "value": {
             "data_type": "person",
             "message_generate_key": "v0.id",
-            "topic": "s\"mc-person-master-${tenant}\""
```
yeah prefer that we use the standard format here now :)
